File: src/main/java/com/tobe/healthy/lessonhistory/repository/LessonHistoryCommentRepository.java
package com.tobe.healthy.lessonhistory.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.tobe.healthy.lessonhistory.domain.entity.LessonHistoryComment;

@Repository
public interface LessonHistoryCommentRepository extends JpaRepository<LessonHistoryComment, Long>,
	LessonHistoryCommentRepositoryCustom {
}

File: src/main/kotlin/com/tobe/healthy/lessonhistory/repository/LessonHistoryCommentRepository.kt
package com.tobe.healthy.lessonhistory.repository

import com.tobe.healthy.lessonhistory.domain.entity.LessonHistoryComment
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface LessonHistoryCommentRepository : JpaRepository<LessonHistoryComment, Long>, LessonHistoryCommentRepositoryCustom
